blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/Instrumentation.java

```diff
@@ -70,19 +69,19 @@
 
     private Instrumentation() {/* Used for JMX exposure */}
 
-    public static Timer.Context getReadTimerContext(ColumnFamily queryCF) {
+    public static Timer.Context getReadTimerContext(String queryCF) {
         return readTimers.getTimerContext(queryCF, false);
```

Comment:
Nope. The API exposes string as the Column Family name. It does have an internal representation of Column Family (which is called Table). The API is very close to JDBC.